src/front-end/src/components/MentorForm.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input, DatePicker, Tooltip, Select } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { InfoCircleOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
+
+const { RangePicker } = DatePicker;
+
+const { Option } = Select;
+
+class MentorForm extends Component {
+  render() {
+    return (
+    <div className="MentorForm"> 
+      <Form>
+        <Form.Item
+          // TEST ITEM (will be deleted):
+          label="Organization"
+          name="Organization"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input
+            placeholder="Please enter your organization"
+            prefix={
+              // tooltip test:
+              <Tooltip title="Organization">
+                <InfoCircleOutlined style={{ color: 'rgba(0,0,0,.45)' }} />
+              </Tooltip>

```

Comment:
I agree that it'd be useful to have an Organization field, and to use a Tooltip to elaborate on what this field might mean. Can you add something more descriptive to the tooltip to describe the field (such something along the lines of: 'Please type the organization that you're planning to be a Mentor with')?

Reply:
done

Reply:
What you updated is the placeholder text, not the tooltip. On this conversation, I'm of the opinion to remove the tooltip, and just leave the placeholder text the way you have it to convey the info we need.